msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -206,6 +208,12 @@ protected override string GenerateCommandLineCommands ()
 				foreach (var nr in NativeReferences)
 					args.AddQuotedLine ("/native-reference:" + Path.GetFullPath (nr.ItemSpec));
 			}
+
+			if (AllNoEmbedBindingProjectNames != null) {
+				foreach (var nr in AllNoEmbedBindingProjectNames)
+					args.AddQuotedLine ("/binding-project-no-embedding:" + Path.GetFullPath (nr.ItemSpec));
```

Comment:
again, I'm not sure `mmp` (or `mtouch`) really need to know about this
it makes then harder to use from the command line (not that's its easy, like it used to, anymore) :|
I'd like to see numbers (maybe there's some win that I'm just missing)